payments-core/src/test/java/com/stripe/android/payments/PaymentIntentFlowResultProcessorTest.kt

```diff
@@ -193,9 +192,54 @@ internal class PaymentIntentFlowResultProcessorTest {
             )
         }
 
+    @Test
+    fun `timeout uses last known response`() {
```

Comment:
This test fails if we use `withTimeout` or remove the `withTimeoutOrNull` block.